src/app/payment/views/IntegratedCheckoutView/CheckoutViewWrapper.tsx

```diff
@@ -225,20 +206,20 @@ const CheckoutViewWrapper = () => {
       });
 
     handleFetchSelectedPlan(planId, currencyValue)
-      .then(async (plan) => {
-        if (checkoutTheme && plan) {
+      .then(async (price) => {
```

Comment:
I think it would be better to change the name of this variable, as `price.price` is not very intuitive, you can also add the type of the argument